core/services/workflows/engine.go

```diff
@@ -540,10 +540,10 @@ func (e *Engine) handleStepUpdate(ctx context.Context, stepUpdate store.Workflow
 	return nil
 }
 
-func (e *Engine) queueIfReady(state store.WorkflowExecution, step *step) {
+func (e *Engine) queueIfReady(state store.WorkflowExecution, s *step) {
```

Comment:
I find this change quite confusing, it's easy for me to get lost when there's single character vars in a large function. Can we keep it as-is?